### PR TITLE
Add support for running Chrome in Docker

### PR DESCRIPTION
**Pull request by jgraham**

### CODE REVIEW

**Review by gsnedders — APPROVED** [review]
This does essentially mean there's very little being contained in the container, right? But, uh, sure?

**Comment by jgraham** [comment]
Well it's better than running with `--privileged` and possibly better than running with `--no-sandbox`. Although I guess if @Hexcles (or someone) claims the latter is better than allowing the two extra syscalls, then I can drop this. But this is how ~everyone seems to approach getting Chrome to run (c.f. https://blog.jessfraz.com/post/how-to-use-new-docker-seccomp-profiles/ which takes a somewhat less minimal approach).

File: tools/ci/run_tc.py [hunk 1]
```diff
@@ -216,6 +216,8 @@ def start_xvfb():
 def get_extra_jobs(event):
     body = None
     jobs = set()
+    if not event:
+        return jobs
```
Comment (Hexcles) [inline tools/ci/run_tc.py]: IIUC, you're trying to make `run_tc.py` work locally, without the various env vars?

I'm not sure if that's a good idea. Isn't `tools/docker` supposed to be the entrypoint for that use case?

**Review by Hexcles — APPROVED** [review]
LGTM % a question

I think it's a good idea to take this PR.

My fix (bc8345181dc) only adds `--no-sandbox` on Taskcluster (when `TASKCLUSTER_ROOT_URL` is present). If anyone tries to run wpt inside Docker locally, they'd still run into the sandbox issue. This PR should solve that use case (although it does create a discrepancy between the two Docker environments).

Once Taskcluster allows us to customize seccomp profiles, we can switch to this approach on Taskcluster as well.

File: tools/ci/run_tc.py (hunk 1 above)
Comment (jgraham) [reply tools/ci/run_tc.py]: Well I found it pretty useful for it to run locally when testing this and various other things. It's pretty convenient to be able to run a single command and have all the chrome install and Xvfb stuff taken care of.

**Pull request merged**